Fixes #3938, HTML broken into code snippet (#3941)
MicrosoftDocs/windows-itpro-docs: devices/surface/step-by-step-surface-deployment-accelerator.md

    ![The installatin progress window](images/sdasteps-fig5-installwindow.png "The installatin progress window")

    *Figure 5. The Installation Progress window*
    >[!NOTE]

>     >The following error message may be hit while Installing the latest ADK or MDT: "An exception occurred during a WebClient request.". This is due to incompatibility between SDA and BITS. Here is the workaround for this:

Reviewer: ```suggestion
    >This error message may occur while installing the latest version of ADK or MDT: "An exception occurred during a WebClient request." This is due to incompatibility between SDA and BITS. To work around this issue:
```

Author: Testing with VS Code's preview addon, it looks like it might be better to just reword this so it's not contained in a note block. The spacing causes odd line breaks within the code block

Author: Commit https://github.com/MicrosoftDocs/windows-itpro-docs/pull/3941/commits/67c0d6b38271af0d3777ad8aa62a225771fef8b4 alters the note into an optional subsection, similar to other subsections in other steps.

> 

Reviewer: ```suggestion
> 
```

>     In the following two PowerShell scripts:

Reviewer: ```suggestion
   >  In the following two PowerShell scripts:
```

>     ```powershell

Reviewer: ```suggestion
   >  ```powershell
```

>     ```powershell

Reviewer: ```suggestion
    ```PowerShell
```

    %ProgramFiles%\Microsoft\Surface\Deployment Accelerator\Data\PowerShell\Install-MDT.ps1
    %ProgramFiles%\Microsoft\Surface\Deployment Accelerator\Data\PowerShell\INSTALL-WindowsADK.ps1
    ```

    Edit the $BITSTransfer variable in the input parameters to $False as shown below:

>     ```powershell

Reviewer: ```suggestion
    ```PowerShell
```

    Param(
        [Parameter(
            Position=0,
            Mandatory=$False,
            HelpMessage="Download via BITS bool true/false"
        )]
        [string]$BITSTransfer = $False
    )
    ```

8. When the SDA process completes the creation of your deployment share, a **Success** window is displayed. Click **Finish** to close the window. At this point your deployment share is now ready to perform a Windows deployment to Surface devices.

    ### Optional: Create a deployment share without an Internet connection

>     If you are unable to connect to the Internet with your deployment server, or if you want to download the Surface drivers and apps separately, you can specify a local source for the driver an app files at the time of deployment share creation. On the **Configure** page of the SDA wizard, select the **Copy from a Local Directory** check box, as shown in Figure 6. The **Download from the Internet** check box will be automatically deselected. Enter the folder location where you have placed the driver and app files in the **Local Path** field, as shown in Figure 6.

Reviewer 2: ```suggestion
    If you are unable to connect to the Internet with your deployment server, or if you want to download the Surface drivers and apps separately, you can specify a local source for the driver and app files at the time of deployment share creation. On the **Configure** page of the SDA wizard, select the **Copy from a Local Directory** check box, as shown in Figure 6. The **Download from the Internet** check box will be automatically deselected. Enter the folder location where you have placed the driver and app files in the **Local Path** field, as shown in Figure 6.
```

    >[!NOTE]
    >You can format your USB drive with FAT32 from Disk Management, but you must still use DiskPart to set the partition as active for the drive to boot properly.

    After you have prepared the USB drive for boot, the next step is to generate offline media from the SDA deployment share. To create this media, follow these steps:

    1.  Open the **Deployment Workbench** from the **Microsoft Deployment Toolkit** group on your Start screen.

    2.  Expand the **Deployment Shares** node and the **Microsoft Surface Deployment Accelerator** deployment share.

    3.  Expand the folder **Advanced Configuration** and select the **Media** folder.

>     4.  Right-click the **Media** folder and click **New Media** as shown in Figure 8 to start the New Media Wizard.

Reviewer 2: ```suggestion
4.  Right-click the **Media** folder and click **New Media** as shown in Figure 8 to start the New Media Wizard.
```

    13. Use your mouse to highlight all of the text displayed in the text box of the **Rules** tab, and then press **Ctrl+V** to paste the text you copied from the **Microsoft Surface Deployment Accelerator** deployment share rules.

    14. Right-click the **Microsoft Surface Deployment Accelerator** deployment share folder, click **Properties**, and then click the **Rules** tab again. Click the **Bootstrap.ini** button to open Bootstrap.ini in Notepad.

    15. Press **Ctrl+A** to select all of the text in the window, and then press **Ctrl+C** to copy the text.

    16. Close Bootstrap.ini and click **OK** in **Microsoft Surface Deployment Accelerator** deployment share properties to close the window.

    17. Right-click the newly created **MEDIA001** item in the **Media** folder, click **Properties**, and then click the **Rules** tab again. Click the **Bootstrap.ini** button to open Bootstrap.ini in Notepad.

    18. Press **Ctrl+A** to select all of the text in the window, then press **Ctrl+V** to paste the text from the SDA deployment share Bootstrap.ini file.

    19. Delete the following lines from the Bootstrap.ini as shown in Figure 11, and then save the file:

>         ```powershell

Reviewer 2: ```suggestion
        ```PowerShell
```
